packages/wrangler/src/pipelines/index.ts

```diff
@@ -136,9 +141,20 @@ function addCreateAndUpdateOptions(yargs: Argv<CommonYargsOptions>) {
 			type: "string",
 			demandOption: false,
 		})
+		.option("binding", {
+			describe: "Enable Worker binding to this pipeline\nDefault: true",
```

Comment:
`yargs` comes with a `default` option config that you can set (see my [suggestion](https://github.com/cloudflare/workers-sdk/pull/6970/files#r1804558117) below), which makes explicitly specifying the default in your description unnecessary.

```suggestion
			describe: "Enable Worker binding to this pipeline",
```

Reply:
this is how it'll look like

<img width="785" alt="Screenshot 2024-10-17 at 12 00 28" src="https://github.com/user-attachments/assets/6319b8fe-5606-4dc5-a4b6-f295e006b063">